csharp/src/Microsoft.ML.OnnxRuntime/FixedBufferOnnxValue.shared.cs

```diff
@@ -39,6 +44,22 @@ public static FixedBufferOnnxValue CreateFromTensor<T>(Tensor<T> value)
             return new FixedBufferOnnxValue(ref ortValue, OnnxValueType.ONNX_TYPE_TENSOR, elementType);
         }
 
+#if NET8_0_OR_GREATER
+#pragma warning disable SYSLIB5001 // System.Numerics.Tensors is only in preview so we can continue receiving API feedback
+        /// <summary>
+        /// Creates a <see cref="FixedBufferOnnxValue"/> object from the tensor and pins its underlying buffer.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="value"></param>
+        /// <returns>a disposable instance of FixedBufferOnnxValue</returns>
+        public static FixedBufferOnnxValue CreateFromDotnetTensor<T>(DotnetTensors.Tensor<T> value) where T : unmanaged
```

Comment:
`FixedBufferOnnxValue` is to be deprecated, so I suggest we do not add any further support for that.
`OrtValue` interfaces are the way to go.

Reply:
Removed